lib/components/form/util.tsx

```diff
@@ -69,3 +72,45 @@ export const setModeButton =
       encodeQueryParams(modesQueryParamConfig, { modeButtons: newButtons })
     )
   }
+
+export const alertUserTripPlan = (
+  intl,
+  currentQuery,
+  onPlanTripClick,
+  routingQuery
```

Comment:
getting eslint warnings and errors for no return type on function and implicit 'any' types.